README.md

```diff
@@ -12,10 +15,12 @@
 npm install -g asyncapi-generator
 ```
 
-Or use all the commands below using Docker by replacing `ag` in all the commands below with:
+Or just use Docker:
 
 ```bash
-docker run --rm -it -v $PWD:/app -w /app asyncapi/generator [COMMAND HERE]
+docker run --rm -it -v [ASYNCAPI FILES LOCATION]:/app/my_files asyncapi/generator [COMMAND HERE]
+# Example that you can run inside generator directory after cloning this repository
+docker run --rm -it -v ${PWD}/test/docs:/app/my_files asyncapi/generator -o ./my_files my_files/streetlights.yml markdown
```

Comment:
I guess the common use case would be `use this AsyncAPI file and generate the output to some folder` therefore if we want to include an example of how to use the docker image, I think we have to split the volume for both the AsyncAPI file and the output, even for the custom template. 
If we just want to use a custom AsyncAPI file:
```bash
docker run --rm -it \
-v /my/output/path:/app/output \
-v /my/spec/path/to/file.yml:/app/asyncfile.yml \
asyncapi/generator -o ./output app/asyncfile.yml markdown
```
If we want to use it with a custom template called `CustomTemplate`:
```bash
docker run --rm -it \
-v /my/output/path:/app/output \
-v /my/spec/path/to/file.yml:/app/asyncfile.yml \
-v /my/template/path:/app/custom_template \
asyncapi/generator --templates app/templates -o ./output app/asyncfile.yml CustomTemplate
```
However the complexity increases all of the sudden, and we move away from a nice and easy one liner 😄 
What you think @derberg ?

Reply:
I like the idea of a split between input and output, it looks clearer now I think - have a look

as it comes to the custom template example, don't you think it might be a bit to much? First of all example will not work by copy/paste, and I would kind of expece that main example already clearly introduces the concept that should be easy to modify, don't you think?

Reply:
Yes, the problem/solution is the same as for the AsyncAPI file, so I agree we can leave the template parameter out.  

Looks fine 👍